tests/Sdk/Unit/Trace/SpanResultTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Unit\Trace;
+
+use OpenTelemetry\Sdk\Trace\Attributes;
+use OpenTelemetry\Sdk\Trace\SamplingResult;
+use OpenTelemetry\Trace\Links;
+use PHPUnit\Framework\TestCase;
+
+class SpanResultTest extends TestCase
+{
+    /**
+     * @dataProvider provideAttributesAndLinks
+     */
+    public function testAttributesAndLinksGetters($attributes, $links)
+    {
+        $result = new SamplingResult(SamplingResult::NOT_RECORD, $attributes, $links);
+
+        $this->assertSame($attributes, $result->getAttributes());
+        $this->assertSame($links, $result->getLinks());
+    }
+
+    /**
+     * Provide different sets of data to test SamplingResult constructor and getters
+     */
+    public function provideAttributesAndLinks(): array
+    {
+        return [
+            [
+                new Attributes(['foo' => 'bar']),
+                $this->createMock(Links::class),
```

Comment:
I haven't found Links implementation

Reply:
Are you referring to [this](https://github.com/open-telemetry/opentelemetry-php/blob/master/api/Trace/Links.php) links implementation?

Reply:
Yes